docs/UsersGuide/user/cmd-evt-chn-prm.md

```diff
@@ -100,8 +100,8 @@ component. This function expects an argument to be supplied for each argument de
 automatically adds ports for retrieving a time tag and sending events. There are two independent ports for sending
 events:
 
-1. binary log output port for sending outside the system
-2. a text log output port for on-board consoles
+1. binary log output port for on-board consoles
```

Comment:
This line is mistaken. The binary log output goes to the ground (off system), while "text log" goes to the system console